server_test.go

```diff
@@ -846,14 +846,14 @@ func TestServer_ExecuteWildcardQuery(t *testing.T) {
 	s.CreateUser("susy", "pass", false)
 
 	// Write series with one point to the database.
```

Comment:
Let's update this comment so it state the system deliberately writes 1 point per time, to guarantee ordering on the output.